### PR TITLE
Update the docket_sheets_link.feature test

### LINKED ISSUE #188: Update docket_sheets_link.feature
Update so test passes with latest test framework.

### PR DESCRIPTION
**Pull request by niharikasingh**
Closes #188

### CODE REVIEW

File: tests/features/docket_sheets_link.feature [hunk 1]
```diff
@@ -21,35 +21,43 @@ Feature: Link to docket sheets pages flow
...
+    Given I start the interview at "entrypoint-petitioner"
+    Then the question id should be "intro"
+    When I tap to continue
+    Then the question id should be "in_ma"
+    When I set the var "has_ma_juv_record" to "True"
+    And I tap the defined text link "docket sheets"
+    When I tap the link "Read more about docket sheets…"
+    Then the question id should be "what_are_docket_sheets"
+    When I set the var "wants_docket_instructions" to "False"
     Then I should see the phrase "Do you already have certified copies"
```
Comment (knod) [inline tests/features/docket_sheets_link.feature:33]: We're replacing phrases with IDs

File: tests/features/docket_sheets_link.feature [hunk 2]
```diff
@@ -21,35 +21,43 @@ Feature: Link to docket sheets pages flow
...
+    When I tap to continue
+    Then the question id should be "in_ma"
+    When I set the var "has_ma_juv_record" to "True"
+    And I tap the defined text link "docket sheets"
+    When I tap the link "Read more about docket sheets…"
+    Then the question id should be "what_are_docket_sheets"
+    When I set the var "wants_docket_instructions" to "True"
     Then I should see the phrase "HOW TO GET CERTIFIED DOCKET SHEETS"
-    And I click the button "I want to keep going with the form "
+    Then the question id should be "do_you_have_sheets"
```
Comment (knod) [inline tests/features/docket_sheets_link.feature:46]:
```
✖ Then the question id should be "do_you_have_sheets" # node_modules/docassemble-cucumber/lib/steps.js:265
       AssertionError: ~ This page's actual question id is "certified-copies-page-text-question"
```

**Review by niharikasingh — COMMENTED** [review]
The tests for this are failing. Pourquoi?!

File: tests/features/docket_sheets_link.feature [hunk 3]
```diff
@@ -21,35 +21,43 @@ Feature: Link to docket sheets pages flow
...
+    When I tap to continue
+    Then the question id should be "in_ma"
+    When I set the var "has_ma_juv_record" to "True"
+    And I tap the defined text link "docket sheets"
+    When I tap the link "Read more about docket sheets…"
+    Then the question id should be "what_are_docket_sheets"
+    When I set the var "wants_docket_instructions" to "True"
     Then I should see the phrase "HOW TO GET CERTIFIED DOCKET SHEETS"
-    And I click the button "I will come back to the form after ordering my docket sheets "
+    Then the question id should be "do_you_have_sheets"
```
Comment (knod) [inline tests/features/docket_sheets_link.feature:61]:
```
✖ Then the question id should be "do_you_have_sheets" # node_modules/docassemble-cucumber/lib/steps.js:265
       AssertionError: ~ This page's actual question id is "certified-copies-page-text-question"
```

**Review by knod — CHANGES_REQUESTED** [review]
Good progress, a few changes

**Review by rpigneri-vol — APPROVED** [review]

**Pull request merged**